untitled1/src/main/com/aca/homework/week8/random/generator/Result.java

```diff
@@ -0,0 +1,24 @@
+package com.aca.homework.week8.random.generator;
+
+public class Result {
+    private String resultText;
+    private long executionDurationInMillis;
+
+    public Result(String resultText, long executionDurationByMillis) {
+        this.resultText = resultText;
+        this.executionDurationInMillis = executionDurationByMillis;
+    }
+
+    public long getExecutionDurationByMillis() {
+        return executionDurationInMillis;
+    }
+
+    public String getResultText() {
+        return resultText;
+    }
+
+    public static Result resultOf(String resultText, long executionDurationByMillis) {
```

Comment:
here there is no need to create a static factory because it is not better than your the constructor